### PR TITLE
setcap for daemon_cl

### PR DESCRIPTION
**Pull request by keerockl**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -108,8 +108,8 @@ endif()
 #------------------------------------------------------------------
 #Install
 #------------------------------------------------------------------
-
 install(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/deps/gptp/linux/build/obj/daemon_cl DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(CODE "execute_process(COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/setcap.sh ${CMAKE_INSTALL_PREFIX}/bin/daemon_cl)")
```
Comment (edersondisouza) [inline CMakeLists.txt]: Maybe use `${CMAKE_INSTALL_BINDIR}/daemon_cl`?

Comment (keerockl) [reply CMakeLists.txt]: it doesn't take the variable in the context.

running setcap on PATH env var
Failed to set capabilities on file `bin/daemon_cl' (No such file or directory)

Comment (edersondisouza) [reply CMakeLists.txt]: You should probably use the `_FULL` version of the variable then:  CMAKE_INSTALL_FULL_BINDIR`.

**Review by edersondisouza — COMMENTED** [review]
Looking ok. Squash the commits?

**Pull request merged**